handlers/missing_test/__init__.py

```diff
@@ -14,7 +14,7 @@ class MissingTestHandler(EventHandler):
     def on_pr_opened(self, api, payload):
         components_changed = set()
 
-        for line in api.get_diff_headers():
+        for line in api.get_changed_files():
```

Comment:
These should probably be changed to read `for filepath in api.get_changed_files():` for clarity.